docs/admin/access.rst

```diff
@@ -79,46 +69,47 @@ Project access control
 .. note::
 
     By turning on access control, all users are prohibited from accessing anything
-    within a given project, unless you add the permissions for them to do just that.
+    within the given project, unless you add the permissions for them to do just that.
 
 .. include:: /snippets/not-hosted-libre.rst
 
 Limit user's access to individual projects by selecting a different access
-control variation on the :guilabel:`Access` tab in the :guilabel:`Settings` of
-each respective project.  This automatically creates several groups for the
+control type from the :guilabel:`Access` tab in the :guilabel:`Settings` of
+each respective project. This automatically creates several groups for the
 project in question, see :ref:`groups`.
 
 :guilabel:`Access control` can be set to:
 
 Public
     Publicly visible, translatable for all logged-in users
 Protected
-    Publicly visible, but translatable only for selected users
+    Publicly visible, but only translatable for selected users
 Private
-    Visible and translatable only for selected users
+    Only visible and translatable for selected users
 Custom
-    Django admin manages users instead of Weblate, see :ref:`custom-acl`.
+    The Django admin-interface manages users instead of Weblate, see :ref:`custom-acl`.
 
 .. image:: /images/project-access.png
 
-Grant access to a project by adding the privilege either directly to an user,
-or group of users in the Django admin-interface, or by using user management on
-the project page, as described in :ref:`manage-acl`.
+Grant access to a project by adding the privilege either directly to a user,
+or group of users in the Django admin-interface, or by using `User management`
+from the project page in question, as described in :ref:`manage-acl`.
 
 .. note::
 
     Even with access control turned on, some info will be available about your
     project:
 
-    * Statistics for the whole instance, including counts for all projects.
-    * Language summary for the whole instance, including counts for all projects.
+    * Statistics for the whole site, including the number of projects on it.
```

Comment:
What are counts? Tallies? People that live in castles?

Reply:
For the record, first time I've stumbled upon the term "tallies" in Webblate, I was very confused what is it and felt strong urgent to replace it with `counts`...

I don't suggest to change it... just wanted to share my experience...